resources/api/openapi.yml

```diff
@@ -159,6 +190,67 @@ components:
                     type: string
                     example: "Oryx Figurine"
 
+        TransfersRequest:
+            properties:
+                orderItems:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/OrderItem'
+                    minItems: 1
+
+        TransfersResponse:
+            properties:
+                transfers:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/Transfer'
+        Transfer:
+            properties:
+                merchantReference:
+                    type: string
+                orderReference:
+                    type: string
+                amount:
+                    type: string
+                transferId:
+                    type: string
+                    description: 'Will be returned to the Tenant so that he knows about the transaction ID and to use the transaction ID for a reverse payout.'
+                orderItems:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/OrderItem'
+                isSuccessful:
+                    type: boolean
+                    description: 'Indicates if the transfer was successful or not.'
+                failureMessage:
+                    type: string
+                    description: 'Message in case of failure to know the issue on the Tenant side.'
+        OrderItem:
+            properties:
+                transferId:
+                    type: string
+                    description: 'ID of a previous made transfer, passed to make a reverse transfer.'
+                merchantReference:
+                    type: string
+                    description: 'UUID of the merchant.'
+                orderReference:
+                    type: string
+                    description: 'UUID of the sales order.'
```

Comment:
orders and merchant references are not [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) (Universally Unique Identifier). So just ID